crmsh/sbd.py

```diff
@@ -184,7 +200,7 @@ def _determine_sbd_watchdog_timeout(self):
         if utils.is_qdevice_configured() and utils.service_is_active("corosync-qdevice.service"):
             qdevice_sync_timeout = utils.get_qdevice_sync_timeout()
             self._sbd_watchdog_timeout = qdevice_sync_timeout + 5
```

Comment:
@gao-yan So here my question is, should we `plus`?
```Python
self._sbd_watchdog_timeout += qdevice_sync_timeout + 5
```
Or
```Python
if self._sbd_watchdog_timeout < qdevice_sync_timeout:
    self._sbd_watchdog_timeout = qdevice_sync_timeout + 5
```

Reply:
The latter makes sense. And let's provide a "notice" messages so that users know the fact and the reason that a different value from their definition is used.

Reply:
Changed as
```Python
-            self._sbd_watchdog_timeout = qdevice_sync_timeout + 5
+            if self._sbd_watchdog_timeout <= qdevice_sync_timeout:
+                watchdog_timeout_with_qdevice = qdevice_sync_timeout + 5
+                bootstrap.warn("sbd_watchdog_timeout is set to {} for qdevice, it was {}".format(watchdog_timeout_with_qdevice, self._sbd_watchdog_timeout))
+                self._sbd_watchdog_timeout = watchdog_timeout_with_qdevice
```